docs/EN_US/ECLWatch/TheECLWatchMan.xml

```diff
@@ -64,49 +73,98 @@
-    your <!-- DNT-Start -->ESP<!-- DNT-End -->/<!-- DNT-Start -->ECL<!-- DNT-End --> Watch node's <!-- DNT-Start -->IP<!-- DNT-End --> address. This will launch <!-- DNT-Start -->ECL<!-- DNT-End --> Watch.</para>
-
-    <para><!-- DNT-Start -->ECL<!-- DNT-End --> Watch is organized by categories based on several system
-    functions. These functions or actions are accessible through the links
-    along the top of the main home page. However there are several <!-- DNT-Start -->ECL<!-- DNT-End --> Watch
+    <para>The <!-- DNT-Start -->ECL<!-- DNT-End --> Watch interface is a
+    browser based set of pages where you can access and interface with the
+    <!-- DNT-Start -->HPCC<!-- DNT-End --> System. To Run <emphasis
+    role="bold"><!-- DNT-Start -->ECL<!-- DNT-End --> Watch</emphasis> using
+    you browser, go to the node on your system running <emphasis
+    role="bold"><!-- DNT-Start -->ESP<!-- DNT-End -->/<!-- DNT-Start -->ECL<!-- DNT-End -->
+    Watch</emphasis> on port 8010. For example, http://nnn.nnn.nnn.nnn:8010,
+    where nnn.nnn.nnn.nnn is your <!-- DNT-Start -->ESP<!-- DNT-End -->/<!-- DNT-Start -->ECL<!-- DNT-End -->
+    Watch node's <!-- DNT-Start -->IP<!-- DNT-End --> address. This will
+    launch <!-- DNT-Start -->ECL<!-- DNT-End --> Watch.</para>
+
+    <para><!-- DNT-Start -->ECL<!-- DNT-End --> Watch is organized by
+    categories based on several system functions. These functions or actions
+    are accessible through the links along the top of the main home page.
+    However there are several <!-- DNT-Start -->ECL<!-- DNT-End --> Watch
     items with common functionality universally across pages of <!-- DNT-Start -->ECL<!-- DNT-End -->
     Watch.</para>
 
     <sect1 id="Common_Elements">
-      <!-- DNT-Start --><title>Common Elements</title><!-- DNT-End -->
+      <!-- DNT-Start -->
+
+      <title>Common Elements</title>
+
+      <!-- DNT-End -->
 
       <para>There are common elements that are useful anywhere within <!-- DNT-Start -->ECL<!-- DNT-End -->
       Watch.</para>
 
+      <sect2>
+        <title>User Login</title>
+
+        <para>When you open ECL Watch a log in window displays. <figure>
+            <title>Login</title>
+
+            <mediaobject>
+              <imageobject>
+                <imagedata fileref="images/ECLWAlogin.jpg" vendor="ECLWatchSS" />
+              </imageobject>
+            </mediaobject>
+          </figure></para>
+
+        <para>You login with the credentials supplied by your Systems
+        Administrator. After a configurable period of inactivity your ECL
+        Watch session locks and you need to enter your credentials to unlock
+        and resume your session. The session remains active for as long as
+        there is regular interaction.</para>
+
+        <para>Additionally, there is a Logout menu item available you can
+        close your session when you complete your work.</para>
```

Comment:
Additionally, there is a Logout menu item available you can "CLICK TO" close your session when you complete your work